source/features/avoid-accidental-submissions.tsx

```diff
@@ -0,0 +1,39 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function addQuickSubmit(): void {
+	select('input#commit-summary-input')?.classList.add('js-quick-submit');
```

Comment:
Why `?.`? When is this missing?

Reply:
It missed my attention in the copy-paste. Changed to `!`.